src/5e-SRD-Features.json

```diff
@@ -1,8777 +1,8666 @@
-    "level": 15,
-    "desc": [
-      "Beginning at 15th level, your rage is so fierce that it ends early only if you fall unconscious or if you choose to end it."
-    ],
-    "url": "/api/features/persistent-rage"
-  },
-  {
-    "index": "barbarian-ability-score-improvement-4",
-    "class": {
-      "index": "barbarian",
-      "name": "Barbarian",
-      "url": "/api/classes/barbarian"
-    },
-    "subclass": {},
-    "name": "Barbarian: Ability Score Improvement 4",
-    "level": 16,
-    "desc": [
-      "When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature."
-    ],
-    "url": "/api/features/barbarian-ability-score-improvement-4"
-  },
-  {
-    "index": "brutal-critical-3-dice",
-    "class": {
-      "index": "barbarian",
-      "name": "Barbarian",
-      "url": "/api/classes/barbarian"
-    },
-    "subclass": {},
-    "name": "Brutal Critical (3 dice)",
-    "level": 17,
-    "desc": [
-      "Beginning at 9th level, you can roll one additional weapon damage die when determining the extra damage for a critical hit with a melee attack. This increases to two additional dice at 13th level and three additional dice at 17th level."
-    ],
-    "url": "/api/features/brutal-critical-3-dice"
-  },
-  {
-    "index": "indomitable-might",
-    "class": {
-      "index": "barbarian",
-      "name": "Barbarian",
-      "url": "/api/classes/barbarian"
-    },
-    "subclass": {},
-    "name": "Indomitable Might",
-    "level": 18,
-    "desc": [
-      "Beginning at 18th level, if your total for a Strength check is less than your Strength score, you can use that score in place of the total."
-    ],
-    "url": "/api/features/indomitable-might"
-  },
-  {
-    "index": "barbarian-ability-score-improvement-5",
-    "class": {
-      "index": "barbarian",
-      "name": "Barbarian",
-      "url": "/api/classes/barbarian"
-    },
-    "subclass": {},
-    "name": "Barbarian: Ability Score Improvement 5",
-    "level": 19,
-    "desc": [
-      "When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature."
-    ],
-    "url": "/api/features/barbarian-ability-score-improvement-5"
-  },
-  {
-    "index": "primal-champion",
-    "class": {
-      "index": "barbarian",
-      "name": "Barbarian",
-      "url": "/api/classes/barbarian"
-    },
-    "subclass": {},
-    "name": "Primal Champion",
-    "level": 20,
-    "desc": [
-      "At 20th level, you embody the power of the wilds. Your Strength and Constitution scores increase by 4. Your maximum for those scores is now 24."
-    ],
-    "url": "/api/features/primal-champion"
-  },
-  {
```

Comment:
Why are we removing the features?

Reply:
Ahh, this looks like a mistake by me, fixing now

Reply:
ahh, i had a json auto-formatter enabled that caused a diff on every line when i saved this file

Reply:
I noticed that. :D

Reply:
Looks like we're still removing these features though?

Reply:
Is that on purpose?

Reply:
@ecshreve Your PR seems good, except for the part where you remove the features. If we look at the features file, only the warlock is in the proper state. Could you put the other spellcasting classes back in order and only remove the reference field?

Reply:
Added 2 new commits to this PR, one reverts the commit that removed spellcasting from the features file, and the second fixes the `reference` urls in each of the spellcasting feature blobs to reference the correct `/classes/<class>/spellcasting` url.